Libs/MRML/Widgets/qMRMLLayoutManager.cxx

```diff
@@ -1330,3 +1330,81 @@ QWidget* qMRMLLayoutManager::viewWidget(vtkMRMLNode* viewNode) const
 
   return d->viewWidget(viewNode);
 }
+
+//------------------------------------------------------------------------------
+void qMRMLLayoutManager::pauseRender(bool pause)
+{
+  // Note: views that are instantiated between pauseRender() and resumeRender() calls will
+  // not be affected by the specified pause state
+
+  Q_D(qMRMLLayoutManager);
+
+  qMRMLLayoutManagerPrivate::PausedViewVector currentPausedViews;
+
+  QString sliceNodeType = "vtkMRMLSliceNode";
+  qMRMLLayoutViewFactory* sliceViewFactory = this->mrmlViewFactory(sliceNodeType);
+  foreach(const QString& viewName, sliceViewFactory->viewNodeNames())
+    {
+    ctkVTKAbstractView* view = (ctkVTKAbstractView*)this->sliceWidget(viewName)->sliceView();
```

Comment:
use qobject_cast<> instead of C-style cast

Reply:
That reminds me of something.

This works fine without any casting:
ctkVTKAbstractView* view = this->threeDWidget(viewName)->threeDView();

While this requires some sort of cast to store in ctkVTKAbstractView:
ctkVTKAbstractView* view = this->sliceWidget(viewName)->sliceView();

They both follow the same inheritance. Any idea why the casting is necessary?

Reply:
If the pointers are compatible but there is a compilation error then maybe the header file of the child class is not included.